test/uniswap/v3/uni-create-pool.js

```diff
@@ -0,0 +1,70 @@
+/* eslint-disable max-len */
+/* eslint-disable no-await-in-loop */
+const { expect } = require('chai');
+const hre = require('hardhat');
+const { getAssetInfo } = require('@defisaver/tokens');
+
+const {
+    getProxy, redeploy, fetchAmountinUSDPrice, balanceOf, UNIV3POSITIONMANAGER_ADDR,
+} = require('../../utils');
+
+const { uniV3CreatePool } = require('../../actions.js');
+
+describe('Uni-Mint-V3', () => {
+    let senderAcc;
+    let proxy;
+    let positionManager;
+    const uniPair = {
+        tokenA: 'TORN',
+        tokenB: 'WETH',
+        amount0: fetchAmountinUSDPrice('TORN', '1000'),
+        amount1: fetchAmountinUSDPrice('WETH', '1000'),
+        fee: '500',
+        tickLower: '-120',
+        tickUpper: '120',
+    };
+
+    before(async () => {
+        await redeploy('UniCreatePoolV3');
+        senderAcc = (await hre.ethers.getSigners())[0];
+        proxy = await getProxy(senderAcc.address);
+        positionManager = await hre.ethers.getContractAt('IUniswapV3NonfungiblePositionManager', UNIV3POSITIONMANAGER_ADDR);
+    });
+
+    it('create a pool that does not exist yet', async () => {
```

Comment:
Write a test where it should return some leftovers and check if thats handled properly

Reply:
test already checks if proxy balance before is the same as proxy balance after => meaning that any tokens that weren't sent to uniswap pool are returned to EOA